### PR TITLE
doc: scaleway_compute_instance_volume

### PR DESCRIPTION
**Pull request by DenBeke**

### CODE REVIEW

File: website/docs/r/compute_instance_volume.html.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# scaleway_compute_instance_volume
+
+Creates and manages Scaleway Compute Instance Volumes. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#volumes-7e8a39).
+
+## Example Usage
+
+```hcl
+resource "scaleway_compute_instance_volume" "server_volume" {
+    type       = "l_ssd"
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_volume.html.markdown]: indent

File: website/docs/r/compute_instance_volume.html.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	name       = "some-volume-name"
+	size_in_gb = 20
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `type` - (Optional) Type type of the volume: `b_ssd` (Block SSD), `l_ssd` (Local SSD) or `l_hdd` (Local HDD). Defaults to `b_ssd`.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_volume.html.markdown]: Let's try to use the same format for showing defaut values / optional / required / oneof, this is different than [scaleway_compute_instance_server](https://github.com/terraform-providers/terraform-provider-scaleway/pull/193/files#diff-5c95b28312e3fd1bb991cb7cd9561937R98)

cc @QuentinBrosse

Comment (kindermoumoute) [reply website/docs/r/compute_instance_volume.html.markdown]: Not sure if `l_hdd` should be documented

File: website/docs/r/compute_instance_volume.html.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+sidebar_current: "docs-scaleway-resource-compute-instance-volume"
+description: |-
+  Manages Scaleway Compute Instance Volumes.
+---
+
+# scaleway_compute_instance_volume
+
+Creates and manages Scaleway Compute Instance Volumes. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#volumes-7e8a39).
+
+## Example Usage
```
Comment (QuentinBrosse) [inline website/docs/r/compute_instance_volume.html.markdown]: "Example"

Comment (QuentinBrosse) [inline website/docs/r/compute_instance_volume.html.markdown]: Maybe we could add another example with a server attachment?

File: website/docs/r/compute_instance_volume.html.markdown (hunk 2 above)
Comment (QuentinBrosse) [inline website/docs/r/compute_instance_volume.html.markdown]: "Type type"

Comment (QuentinBrosse) [reply website/docs/r/compute_instance_volume.html.markdown]: > Let's try to use the same format

Yes, we should take [this syntax](https://github.com/terraform-providers/terraform-provider-scaleway/pull/193/files#diff-5c95b28312e3fd1bb991cb7cd9561937R143).

> Not sure if l_hdd should be documented

I'm agree

Comment (kindermoumoute) [reply website/docs/r/compute_instance_volume.html.markdown]: > I'm agree

👋 Hi agree, nice to meet you.

Comment (DenBeke) [reply website/docs/r/compute_instance_volume.html.markdown]: > 👋 Hi agree, nice to meet you.

Missed the opportunity to make a dad joke 😜

File: website/docs/r/compute_instance_volume.html.markdown (hunk 3 above)
Comment (DenBeke) [reply website/docs/r/compute_instance_volume.html.markdown]: Wouldn't it make more sense to link to the server doc? That way we only have to maintain one page when things change...

File: website/docs/r/compute_instance_volume.html.markdown [hunk 4]
```diff
@@ -10,41 +10,34 @@ description: |-
...
+    size_in_gb = 20
 }
 ```
 
 ## Arguments Reference
 
 The following arguments are supported:
 
-- `type` - (Optional) Type type of the volume: `b_ssd` (Block SSD), `l_ssd` (Local SSD) or `l_hdd` (Local HDD). Defaults to `b_ssd`.
+- `type` - (Defaults to `b_ssd`) The type of the volume: `b_ssd` (Block SSD), `l_ssd` (Local SSD). Defaults to `b_ssd`.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_volume.html.markdown]: ``Defaults to `b_ssd`.`` left on left.

Comment (DenBeke) [reply website/docs/r/compute_instance_volume.html.markdown]: 👍

File: website/docs/r/compute_instance_volume.html.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    name       = "some-volume-name"
+    size_in_gb = 20
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `type` - (Defaults to `b_ssd`) The type of the volume. the possible values are: `b_ssd` (Block SSD), `l_ssd` (Local SSD).
```
Comment (QuentinBrosse) [inline website/docs/r/compute_instance_volume.html.markdown]: The

File: website/scaleway.erb [hunk 6]
```diff
@@ -70,6 +70,9 @@
             <li<%= sidebar_current("docs-scaleway-resource-compute-instance-ip") %>>
               <a href="/docs/providers/scaleway/r/compute_instance_ip.html">scaleway_compute_instance_ip</a>
             </li>
+            <li<%= sidebar_current("docs-scaleway-resource-compute-instance-volume") %>>
```
Comment (QuentinBrosse) [inline website/scaleway.erb]: Just `<li>`

**Pull request merged**